hushline/templates/settings.html

```diff
@@ -139,24 +139,50 @@ <h3>Change Password</h3>
 
     <div id="email" class="tab-content" role="tabpanel" aria-labelledby="email-tab" hidden>
         <!-- SMTP Settings Section -->
-        <h3>Message Forwarding</h3>
+        <h3>Email Forwarding</h3>
+        <p class="meta">✊ Message fowarding is powered by <a href="https://riseup.net" target="_blank"
```

Comment:
type, "forwarding" not "fowarding"

Reply:
And the header says "Email Forwarding" so the message should probably also say "Email forwarding" instead of "Message forwarding"